modules/dnn/src/layers/detection_output_layer.cpp

```diff
@@ -55,11 +55,7 @@
 
 #ifdef HAVE_DNN_NGRAPH
```

Comment:
HAVE_DNN_NGRAPH => HAVE_DNN_OPENVINO ?

Reply:
Will cost a lot of changes as well as DNN_BACKEND_INFERENCE_ENGINE -> DNN_BACKEND_OPENVINO.